src/index.html

```diff
@@ -4,23 +4,25 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>Leaderboard Project</title>
 </head>
 
 <body>
-    <div class='container'>  
+    <div class='container'>
         <div>
             <h1>Leaderboard</h1>
+            <form>
+                <p>Add Your score</p>
+                <input type='text' id='name' placeholder='Your name' required>
+                <input type='text' id='score' placeholder='Your Score' required>
+                <input type='submit' id='submit' value='Submit'>
+                <p id='error' class='error'></p>
+            </form>
             <p>Recent Scores</p>
             <button id='refresh'>Refresh</button>
-            <div id='leaderboard'></div>
+            <div id='leaderboard' class='leaderboard'></div>
```

Comment:
- [ ] Kindly lets follow the layout of the wireframe given 🙏 . You can change the colors, background, fonts etc. But the positioning of the elements should be the same as from the requirements: 👇 
![image](https://user-images.githubusercontent.com/68167810/132840164-f048170f-50ae-4e38-b93c-0a2e0e00598b.png)

![image](https://user-images.githubusercontent.com/68167810/132840280-27ff323d-0ef1-42ae-8270-d06958587f68.png)

Reply:
Thanks for the Review, I implemented your feedback.